tools/reviewer/webapp/src/app/dashboard/file-changes/code-changes/services/changes.service.ts

```diff
@@ -125,8 +125,9 @@ export class ChangesService {
     this.addPlaceholders(textDiff.getRightChangeList(), rightBlockLines);
 
     if (leftBlockLines.length !== rightBlockLines.length) {
-      // After adding all placeholders
-      throw new Error('Blocks should have the same amount of lines');
+      throw new Error(
+        'After adding all placeholders, blocks should have the same amount of lines',
```

Comment:
Can you add `leftBlockLines.length` and `rightBlockLines.length` to the error so we know which side has less?